src/chapter_2.md

```diff
@@ -0,0 +1,7 @@
+# Chapter 2: Essential Features
+Brickadia has an array of "quality of life" features, if those didn't exist, you wouldn't be able to do things easily.
```

Comment:
Slight run-on scentence here, consider replacing first ","

Reply:
What about something similar this?
> Brickadia has many quality of life features designed to make a lot of common things easier.
I agree that the way it's written now is very odd.

Reply:
sentence must be rewritten like the above